de.klayout.KLayout.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: de.klayout.KLayout
```

Comment:
Please reformat file as described in https://github.com/flathub/flathub/wiki/YAML-Style-Guide

Reply:
The file has been reformatted.